modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/SelectEventEntityInput.es.js

```diff
@@ -51,7 +52,10 @@ class SelectEventEntityInput extends React.Component {
 				const value = JSON.parse(event.value);
 
 				onChange({
-					assetId: value.fileEntryId,
+					assetId: hash
+						.sha256()
+						.update(value.fileEntryId)
+						.digest('hex'),
```

Comment:
Do we really need to use `hash.js`? We can create the same assetId using the native API, it is well supported. This is from MDN (https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#converting_a_digest_to_a_hex_string):

```js
async function digestMessage(message) {
  const msgUint8 = new TextEncoder().encode(message); // encode as (utf-8) Uint8Array
  const hashBuffer = await crypto.subtle.digest("SHA-256", msgUint8); // hash the message
  const hashArray = Array.from(new Uint8Array(hashBuffer)); // convert buffer to byte array
  const hashHex = hashArray
    .map((b) => b.toString(16).padStart(2, "0"))
    .join(""); // convert bytes to hex string
  return hashHex;
}
```

Reply:
Oh I'll take a look, thank you @p2kmgcl !

Reply:
Updated! :smile: